modules/ROOT/pages/questions/question-07.adoc

```diff
@@ -0,0 +1,14 @@
+7. two re-namings happened consequent to `noneditable` being re-cast
```

Comment:
@lnewson

Reply:
This does need to be noted in the migration guide, but the settings are purely renamed. There are no functional changes.

Reply:
>  Socratic question: are these re-named options *Functions* or
   *Commands*?

Neither, these are just plain *Options* which take a string with the class name. For example, `noneditable_class: 'mceNonEditable'`.

> Do these re-namings constitute something an upgrader needs to be
   aware of? That is, does the re-naming need to be noted in the
   _Migration Guide_? And, if so, what does the upgrader need to do to
   cater to the change?

Yes, they'll also need to rename the options used in their TinyMCE init configuration to match the new names.